### PR TITLE
Add BBEs related to Log, UUID, OS, Random modules

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: new-examples/logging-with-context/logging-with-context.bal [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    // You can pass key/value pairs where the values are function pointers.
+    // These functions can return values which change dynamically.
+    // The following log prints the current UTC time as a key/value pair.
+    log:printInfo("info log",
+                  current_time = isolated function() returns string {
+                      return time:utcToString(time:utcNow());});
+    // The following log prints a random percentage as a key/value pair.
+    log:printError("error log",
+                   percentage = isolated function() returns float {
+                       return random:createDecimal() *100.0;});
```
Comment (daneshk) [inline new-examples/logging-with-context/logging-with-context.bal:17]: Let's make this a info, this is not the error log example

Comment (daneshk) [inline new-examples/logging-with-context/logging-with-context.bal:17]:
```suggestion
                       return random:createDecimal() * 100.0;});
```

**Review by daneshk — APPROVED** [review]
LGTM

File: new-examples/environment-variables/environment-variables.bal [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import ballerina/io;
+import ballerina/os;
+
+public function main() {
+    // Returns the environment variable value associated with "HTTP_PORT".
```
Comment (praneesha) [inline new-examples/environment-variables/environment-variables.bal:5]:
```suggestion
    // Returns the environment variable value associated with the `HTTP_PORT`.
```

File: new-examples/environment-variables/environment-variables.metatags [hunk 3]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to to retrieve information about the OS.
+keywords: ballerina, ballerina by examples, bbe, os, environment
```
Comment (praneesha) [inline new-examples/environment-variables/environment-variables.metatags:2]:
```suggestion
keywords: ballerina, ballerina by examples, BBE, OS, environment
```
Please correct in all occurrences.

File: new-examples/logging-configuration/logging-configuration.out [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# Each module can also be assigned its own log level. To assign a
+# log level to a module, provide the following entry in the `Config.toml` file:
+#
+# ```
+# [[ballerina.log.modules]]
+# name = "[ORG_NAME]/[MODULE_NAME]"
+# level = "[LOG_LEVEL]"
+# ```
+
+# By default, log messages are logged to the console in LogFmt format.
```
Comment (praneesha) [inline new-examples/logging-configuration/logging-configuration.out:30]:
```suggestion
# By default, log messages are logged to the console in the LogFmt format.
```

File: new-examples/logging-with-context/logging-with-context.bal [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import ballerina/log;
+import ballerina/random;
+import ballerina/time;
+
+public function main() {
+    // The Ballerina log API provides functions to log at four levels, which are
+    // `DEBUG`, `ERROR`, `INFO`, and `WARN`.
+    // You can pass key/value pairs where the values are function pointers.
+    // These functions can return values which change dynamically.
```
Comment (praneesha) [inline new-examples/logging-with-context/logging-with-context.bal:9]:
```suggestion
    // These functions can return values, which change dynamically.
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**